templates/types/streaming/fastapi/app/api/routers/chat.py

```diff
@@ -39,27 +39,27 @@ async def chat(
                     MetadataFilter(
                         key="private",
                         value="true",
-                        operator="!=",
+                        operator="!=", # type: ignore
                     ),
                     MetadataFilter(
                         key="doc_id",
                         value=doc_ids,
-                        operator="in", # This won't work with a simple vector store since it uses the 'any' operator for that.
+                        operator="any", # type: ignore . This won't work with a simple vector store since it uses the 'any' operator for that.
```

Comment:
now the comment doesn't make sense

Reply:
Ah, sorry. It should be "in" operator - i changed to "any" just to test the simple vector store but forgot to revert it :D